js/APIMiddleware.js

```diff
@@ -0,0 +1,42 @@
+let result;
+/**
+ * API - Description
+ *
+ * @returns {type} Description
```

Comment:
return something thats not the description

Reply:
fixed